### PR TITLE
Task 7: Authorization

### PR DESCRIPTION
**Pull request by nataliagulko**
HW for [Task 7 (Authorization)](https://github.com/EPAM-JS-Competency-center/cloud-development-course-initial/blob/main/7_authorization/task.md)

#### Task 7.1
- `authorization-service` was added
- `basicAuthorizer` lambda was created
-  environment variable `nataliagulko=TEST_PASSWORD` was added
- `basicAuthorizer` decodes basic authorization token
- `basicAuthorizer` lambda returns 403 on invalid authorization_token and 401 HTTP on empty token
- _FYI: `https://8a3sfxli3b.execute-api.eu-west-1.amazonaws.com/dev/protected` was added to play with `basicAuthorizer`_

#### Task 7.2
- `importProductsFile` lambda from import-service uses `basicAuthorizer` for authorization

#### Task 7.3
- FE PR https://github.com/nataliagulko/shop-react-redux-cloudfront/pull/5

### CODE REVIEW

File: authorization-service/serverless.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    stage: "dev",
+    region: "eu-west-1",
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: "1",
+      NODE_OPTIONS: "--enable-source-maps --stack-trace-limit=1000",
+      nataliagulko: "TEST_PASSWORD",
```
Comment (VittalQSC) [inline authorization-service/serverless.ts]: minor: private information should not be visible in code

reference from the task:
Do not send your credentials to the GitHub. Use .env file and serverless-dotenv-plugin serverless plugin to add environment variables to the lambda. Add .env file to .gitignore file.

Comment (nataliagulko) [reply authorization-service/serverless.ts]: Oh, yes, sure. I think, I forgot to remove it! 😱
I have this credentials at .env as well

Thank you!

**Comment by VittalQSC** [comment]
Good job!
Mark: 5

**Comment by rostislav-sergeenkov** [comment]
Reviewed. Total score: 5
Thanks

**Pull request merged**